Kernel/FileSystem/FATFS/FileSystem.cpp

```diff
@@ -86,9 +239,26 @@ Inode& FATFS::root_inode()
     return *m_root_inode;
 }
 
-BlockBasedFileSystem::BlockIndex FATFS::first_block_of_cluster(u32 cluster) const
+Tuple<BlockBasedFileSystem::BlockIndex, size_t> FATFS::first_block_of_cluster(u32 cluster) const
```

Comment:
Tuple outside of generic code is *strongly* discouraged. a small struct with names for the parameters is better for readability in every case we've seen so far.